app/api/search/search.js

```diff
@@ -683,7 +683,8 @@ const buildQuery = async (query, language, user, resources) => {
   queryBuilder.filterMetadata(filters);
   queryBuilder.customFilters(query.customFilters);
   // this is where the query aggregations are built
-  queryBuilder.aggregations(aggregations, dictionaries);
+  const isClasificationEnabled = await dictionariesModel.count({ enable_classification: true });
+  queryBuilder.aggregations(aggregations, dictionaries, isClasificationEnabled);
```

Comment:
This is the problem with comments: the comment that belonged to the `queryBuilder...` line, now is commenting the `isClasification...` (and incorrectly spelled!) line.

Reply:
Resolved in previous commit.